tests/unit/message_status_handler/test_communication_management.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+import requests_mock
+
+from comms_management import get_statuses
+
+class TestCommsManagement(unittest.TestCase):
```

Comment:
If the module we are testing is written in a functional style we should probably do the same style with the test.
In this case it should be as simple as removing line 6 and unindenting the `test_get_statuses` test function.